app/views/cart/show.html.erb

```diff
@@ -15,8 +16,14 @@
       <td>
         <p align = "center">
           <span>
-            <img align = "center" id="cart-image" src= <%= item.image %>>
-            <%=link_to item.name, "/items/#{item.id}" %>
+            <ul><img align = "center" id="cart-image" src= <%= item.image %> style= "max-width:200px;width:100%" ></ul>
```

Comment:
Nice drying up of the code!

Reply:
I'm still struggling with html/css, so I am working on making updates as I see them after as ```save_and_open_page```